staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -266,6 +270,15 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 	return s, nil
 }
 
+// Set list of internal API servers that delegated server supports. It may take
+// some time for these services to get registered with aggregator but
+// we will return them as part of API discovery as they are served
+// regardless of their registration state (all unknown API calls will be
```

Comment:
> they are served regardless of their registration state

The aggregator does not know how the delegated server handles requests, so this not a good assumption